department-of-veterans-affairs/caseflow
# Hp 508 compliance

Connects #5010 

### Description
This PR takes care of the 508 compliance buttons and also corrects the spelling of disposition on the hearing worksheet 

### Acceptance Criteria 
- [ ] The field label for Notes is read as: Time/RO(s) Notes. This label should read as Notes as it appears visually for the field. (6)
- [ ] On the hearing Worksheet page the Save as PDF button cannot be accessed using a keyboard or with assistive technology programs. This button should be included in the tabbing order.
- [ ] On the hearing Worksheet page, the Delete button found on the Current Issues tab cannot be accessed with a keyboard or using assistive technology programs.

### Testing Plan
1. Go to Worksheet and tab on the Trashcan button.
2. Go to the Daily docket and inspect notes to see that it now has a label of notes.
3. Go to the worksheet page and tab on the `Save as PDF button`
